test/connector/http/basicauth/test

```diff
@@ -93,4 +95,8 @@ echo "Test: Secretless configured with incorrect password fails"
 bad_resp=$(ping_target_service_thru_secretless 8081)
 assert_contains "$bad_resp" "401 Unauthorized"
 
-echo "PASS"
+if [[ isError ]]; then
+  exit 1
+else
+  echo "PASS"
+fi
```

Comment:
No need to do the `else` since we can early-exit.
```suggestion
if [[ isError ]]; then
  echo "FAIL: basicauth/test"
  exit 1
fi

echo "PASS: basicauth/test"
```